applications/RomApplication/python_scripts/rom_analysis.py

```diff
@@ -19,9 +19,17 @@ def __init__(self,global_model, parameters):
 
         def _CreateSolver(self):
             """ Create the Solver (and create and import the ModelPart if it is not alread in the model) """
+            self.rom_basis_output_name = 'RomParameters.json' #Default
```

Comment:
Is JSON you primary output format? If yes, I strongly recommend looking for something else.

Reply:
It used to be. Now there are other alternatives available.